editor/multi_node_edit.cpp

```diff
@@ -167,6 +167,8 @@ void MultiNodeEdit::_get_property_list(List<PropertyInfo> *p_list) const {
 	for (List<PLData *>::Element *E = data_list.front(); E; E = E->next()) {
 		if (nc == E->get()->uses) {
 			p_list->push_back(E->get()->info);
+		} else if (E->get()->info.name.begins_with("blend_shapes/")) {
+			p_list->push_back(E->get()->info);
```

Comment:
```suggestion
			// If one of the selected MeshInstance Nodes is missing a single blend shape,
			// all the blendshape sliders would disappear in the Inspector because
			// MultiNodeEdit was expecting a perfect meta info match.
			p_list->push_back(E->get()->info);
```